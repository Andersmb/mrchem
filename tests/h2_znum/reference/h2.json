{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -0.7
          ],
          "radius": 0.79
        },
        {
          "center": [
            0.0,
            0.0,
            0.7
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            -0.7
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            0.7
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 7,
      "basis_type": "legendre",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -4
    },
    "printer": {
      "file_name": "h2.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        }
      },
      "initial_guess": {
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "Hartree-Fock",
        "prec": 0.001,
        "restricted": true,
        "type": "sad",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "energy_thrs": -1.0,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 0.001,
        "helmholtz_prec": -1.0,
        "kain": 3,
        "localize": false,
        "max_iter": 5,
        "method": "Hartree-Fock",
        "orbital_thrs": 0.01,
        "rotation": 0,
        "start_prec": 0.001
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        -1.1189687543466913e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.0002532418251648487,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            0.0,
            0.0,
            0.0002532418251648487
          ],
          "vector_el": [
            0.0,
            0.0,
            0.0002532418251672912
          ],
          "vector_nuc": [
            0.0,
            0.0,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            -0.7
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            0.7
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -0.5959952499172974
        ],
        "occupation": [
          2.0
        ],
        "spin": [
          "p"
        ],
        "sum_occupied": -1.1919904998345947
      },
      "scf_energy": {
        "E_ee": 1.3121426033860593,
        "E_eext": 0.0,
        "E_el": -1.8480644872132097,
        "E_en": -3.6210061253229155,
        "E_kin": 1.116867650731085,
        "E_next": 0.0,
        "E_nn": 0.7142857142857143,
        "E_nuc": 0.7142857142857143,
        "E_tot": -1.1337787729274953,
        "E_x": -0.6560686160074384,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 8,
      "routine": "mrchem.x",
      "total_cores": 8,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 1.178799409192627,
        "E_eext": 0.0,
        "E_el": -1.8126339293755125,
        "E_en": -3.2892103039345457,
        "E_kin": 0.8871016110396834,
        "E_next": 0.0,
        "E_nn": 0.7142857142857143,
        "E_nuc": 0.7142857142857143,
        "E_tot": -1.098348215089798,
        "E_x": -0.5893246456732769,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 1.2655515061916096,
              "E_eext": 0.0,
              "E_el": -1.8437367226175398,
              "E_en": -3.50580292969342,
              "E_kin": 1.0292874138218902,
              "E_next": 0.0,
              "E_nn": 0.7142857142857143,
              "E_nuc": 0.7142857142857143,
              "E_tot": -1.1294510083318254,
              "E_x": -0.6327727129376196,
              "E_xc": 0.0,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -1.1294510083318254,
            "energy_update": 0.03110279324202736,
            "mo_residual": 0.0808961896616029,
            "wall_time": 1.013064049
          },
          {
            "energy_terms": {
              "E_ee": 1.3004666279701727,
              "E_eext": 0.0,
              "E_el": -1.8476537577532146,
              "E_en": -3.591946052295966,
              "E_kin": 1.0940562057368612,
              "E_next": 0.0,
              "E_nn": 0.7142857142857143,
              "E_nuc": 0.7142857142857143,
              "E_tot": -1.1333680434675002,
              "E_x": -0.6502305391642828,
              "E_xc": 0.0,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -1.1333680434675002,
            "energy_update": 0.003917035135674762,
            "mo_residual": 0.02755836969003991,
            "wall_time": 0.808053866
          },
          {
            "energy_terms": {
              "E_ee": 1.3121426033860593,
              "E_eext": 0.0,
              "E_el": -1.8480644872132097,
              "E_en": -3.6210061253229155,
              "E_kin": 1.116867650731085,
              "E_next": 0.0,
              "E_nn": 0.7142857142857143,
              "E_nuc": 0.7142857142857143,
              "E_tot": -1.1337787729274953,
              "E_x": -0.6560686160074384,
              "E_xc": 0.0,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -1.1337787729274953,
            "energy_update": 0.00041072945999509614,
            "mo_residual": 0.008740598342420663,
            "wall_time": 0.804612923
          }
        ],
        "wall_time": 2.625868227
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
