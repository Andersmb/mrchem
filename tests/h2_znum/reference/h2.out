                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         atom-number-input                        ***
***         Git commit hash    c8f3a0ed8d0569970b75-dirty               ***
***         Git commit author  Gabriel                                  ***
***         Git commit date    Mon Mar 8 16:08:41 2021 +0100            ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.0-alpha
 Git branch            : HEAD
 Git commit hash       : 3445540b49faaa09acc3
 Git commit author     : Peter Wind
 Git commit date       : Mon Feb 1 15:42:36 2021 +0100

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : OpenMP (8 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 7
 polynomial type       : Legendre
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000       -0.700000
    1       H            :        0.000000        0.000000        0.700000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999998867959e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : Hartree-Fock
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  0.887101611040
 E-N energy              :            (au)                 -3.289210303935
 Coulomb energy          :            (au)                  1.178799409193
 Exchange energy         :            (au)                 -0.589324645673
 X-C energy              :            (au)                  0.000000000000
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  0.714285714286
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.812633929376
 Nuclear energy          :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.098348215090e+00
                         :      (kcal/mol)             -6.892239052281e+02
                         :        (kJ/mol)             -2.883712821346e+03
                         :            (eV)             -2.988757619979e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.611579721395
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.223159442789
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : Hartree-Fock
 Checkpointing           : Off
 Max iterations          : 5
 KAIN solver             : 3
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-03
 Final precision         : 1.00000e-03
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          1.000000e+00          -1.098348215090         -1.098348e+00
    1          8.089619e-02          -1.129451008332         -3.110279e-02
    2          2.755837e-02          -1.133368043468         -3.917035e-03
    3          8.740598e-03          -1.133778772927         -4.107295e-04
---------------------------------------------------------------------------
                      SCF converged in 3 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000       -0.700000
    1       H            :        0.000000        0.000000        0.700000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.116867650731
 E-N energy              :            (au)                 -3.621006125323
 Coulomb energy          :            (au)                  1.312142603386
 Exchange energy         :            (au)                 -0.656068616007
 X-C energy              :            (au)                  0.000000000000
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  0.714285714286
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.848064487213
 Nuclear energy          :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.133778772927e+00
                         :      (kcal/mol)             -7.114569157632e+02
                         :        (kJ/mol)             -2.976735737485e+03
                         :            (eV)             -3.085169075165e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.595995249917
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.191990499835
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000        0.000253
 Magnitude               :            (au)                        0.000253
                         :         (Debye)                        0.000644
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000000        0.000253
 Magnitude               :            (au)                        0.000253
                         :         (Debye)                        0.000644
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  5s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
